tests/api/test_tableview.py

```diff
@@ -0,0 +1,25 @@
+import json
+from whyis.test.api_test_case import ApiTestCase
+
+class TestTableview(ApiTestCase):
+
+    def test_get(self):
+        graph = "temp"
+        content = self.client.get("/table/" + graph, follow_redirects=True)
+        print(content.json)
+
+    def test_post(self):
+        graph = [
```

Comment:
Use consistent variable names. `graph` is something else in test_get.

I'd suggest you put the test JSON and the table identifier (which you've hard-coded here as `test` and was `graph` above) in class variables.